test/scripts/sendForcedBatch/main.go

```diff
@@ -105,34 +119,34 @@ func sendForcedBatches(cliCtx *cli.Context) error {
 	log.Debug("currentBlock.Time(): ", currentBlock.Time())
 
 	// Get tip
-	tip, err := poe.GetForcedBatchFee(&bind.CallOpts{Pending: false})
+	tip, err := rollupManager.GetForcedBatchFee(&bind.CallOpts{Pending: false})
 	if err != nil {
 		log.Error("error getting tip. Error: ", err)
 		return err
 	}
 
-	// Allow forced batches in smart contract if disallowed
-	disallowed, err := poe.IsForcedBatchDisallowed(&bind.CallOpts{Pending: false})
-	if err != nil {
-		log.Error("error getting isForcedBatchDisallowed. Error: ", err)
-		return err
-	}
-	if disallowed {
-		tx, err := poe.ActivateForceBatches(auth)
-		if err != nil {
-			log.Error("error sending activateForceBatches. Error: ", err)
-			return err
-		}
-		err = operations.WaitTxToBeMined(ctx, ethClient, tx, operations.DefaultTimeoutTxToBeMined)
-		if err != nil {
-
-			log.Error("error waiting tx to be mined. Error: ", err)
-			return err
-		}
-	}
+	// // Allow forced batches in smart contract if disallowed
+	// disallowed, err := zkevm.IsForcedBatchAllowed(&bind.CallOpts{Pending: false})
+	// if err != nil {
+	// 	log.Error("error getting IsForcedBatchAllowed. Error: ", err)
+	// 	return err
+	// }
+	// if disallowed {
+	// 	tx, err := zkevm.ActivateForceBatches(auth)
+	// 	if err != nil {
+	// 		log.Error("error sending activateForceBatches. Error: ", err)
+	// 		return err
+	// 	}
+	// 	err = operations.WaitTxToBeMined(ctx, ethClient, tx, operations.DefaultTimeoutTxToBeMined)
+	// 	if err != nil {
+
+	// 		log.Error("error waiting tx to be mined. Error: ", err)
+	// 		return err
+	// 	}
+	// }
```

Comment:
yes, I've diverged from the upstream here due to being ahead in contract integration. IMO it felt better to leave this commented rather than remove test 😅 . But this is going to be fixed in next batch of pulled changed